.github/CODEOWNERS

```diff
@@ -6,6 +6,7 @@
 
 *                                           @spotify/backstage-core
 /docs/features/techdocs                     @spotify/techdocs-core
+/plugins/cost-insights                      @spotify/silver-lining
```

Comment:
We're gonna be hunting you down a lot with this, since any release etc. will change code here. Wonder if it makes sense to use `/plugins/cost-insights/src`? Could definitely start here anyway and see if we wanna relax it a bit